src/app/components/addMember/addMemberComponent.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { browserHistory, Link } from 'react-router';
+
+import MemberInfoForm from '../memberInfoForm/memberInfoFormContainer';
+
+export default ({ updateMember }) => (
+  <div className="container add-member">
+    <Link to="/"><span className="btn-corner glyphicon glyphicon-remove" /></Link>
+    <h1>Add a team member</h1>
+    <h4 className="subtitle">Set email, location and role.</h4>
+    <div className="divider"></div>
+    <MemberInfoForm initialValues={{role: 'regular'}} onSubmit={ member => { updateMember(member); browserHistory.push('/') } } />
```

Comment:
Can you move all the `browserHistory` calls into the action creators? It's better for the responsibility for changing pages live in Redux. For example, if we connected this component to an API, and the server returned a validation error, we would want to stay on this page and display the error. If the `browserHistory` call is in Redux, we can control this. If it's in the component, the transition will happen even if there's an error.